chcemvediet/apps/anonymization/admin.py

```diff
@@ -0,0 +1,73 @@
+# vim: expandtab
```

Comment:
*Komentar sa tyka binarneho `anonymization.odt`, ku ktoremu sa nedaju davat komentare*

Schema vztahov medzi modelmi je naspravna. `AttachmentNormalization` predsa nema polymorfnu relaciu k lubovolnemu objektu ale FK k `Attachment`. Pls zakresli to tak. Ako za zakresluje FK najdes v `inforequests.odt` (Prazdna gulicka znamena pocetnost 0-1, plna 0+, bez gulicky 1. Tusim to je z [OMT]).

[OMT]: https://en.wikipedia.org/wiki/Object-modeling_technique

Reply:
:ok: